react-native.config.js

```diff
@@ -6,5 +6,10 @@ module.exports = {
         ios: null,
       },
     },
+    'react-native-background-geolocation': {
```

Comment:
Just for reference

https://github.com/react-native-community/cli/blob/main/docs/autolinking.md#how-can-i-disable-autolinking-for-unsupported-library